packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -802,6 +813,10 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 			this.mediaControlsNode = ReactDOM.findDOMNode(node); // eslint-disable-line react/no-find-dom-node
 		}
 
+		getMoreButtonRef = (node) => {
+			this.moreButtonNode = ReactDOM.findDOMNode(node); // eslint-disable-line react/no-find-dom-node
```

Comment:
I am not sure but, ...do we need to use `ReactDOM.findDOMNode()` here?
And don't we need to check if node is equals to `null`?

Reply:
> I am not sure but, ...do we need to use ReactDOM.findDOMNode() here?
Yes, we compare the button Node and the Spotlight.getCurrent() Node in the MediaControlsDecorator, not in the MediaControlsBase.

> And don't we need to check if node is equals to null?
I didn't see the node returning null where refs function.

Reply:
Got it.